### DISCUSSION TITLE
Google search for free?

### DISCUSSION THREAD

**Discussion by Louvivien** [FlowiseAI/Flowise#78] [Ideas]
Is it possible to get the Google search feature without using Serp API?

**Comment by HenryHengZJ** [comment]
Closest one is this - https://js.langchain.com/docs/modules/agents/tools/webbrowser 

There have been requests, will implement this soon!

**Comment by Louvivien** [comment]
Yes they do it on Auto-GPT

**Comment by HenryHengZJ** [comment]
PR merged and pushed to latest release: https://github.com/FlowiseAI/Flowise/pull/88
